add stranded expansion, fix subscript issue (#438)
AlexsLemonade/OpenPBTA-manuscript: content/03.results.md

### Transcriptomic Landscape of Pediatric Brain Tumors

> Since the majority of RNA-Seq samples in the PBTA cohort were prepared with ribosomal RNA depletion followed by stranded sequencing (N = 977) compared to those with poly-A selection (N = 58), the following analyses were performed using stranded samples only.

Reviewer: Quick rephrase - 
```suggestion
The majority of RNA-Seq samples in the PBTA cohort were prepared with ribosomal RNA depletion followed by stranded sequencing (N = 977), while the remaining samples were prepared with poly-A selection (N = 58).
Therefore, the following analyses were performed using stranded samples only.
```

> Since the majority of RNA-Seq samples in the PBTA cohort were prepared with ribosomal RNA depletion followed by stranded sequencing (N = 977) compared to those with poly-A selection (N = 58), the following analyses were performed using stranded samples only.

Reviewer: We might expand this a tad to state plainly that a reason we are only using stranded is because the diagnoses are not evenly distributed across library preps so correcting is a challenge. Can refer to S7A probably as well (the polya/stranded count barplot).

Author: Updated in [f16f692](https://github.com/AlexsLemonade/OpenPBTA-manuscript/pull/438/commits/f16f6929559a96c711afbacb5898d42dbf0514ff) - see if that looks OK to you - trying not to add too much to results at this point :)

Reviewer: This is good! I can usually find more places to cut words when we close to the end of revisions too as needed.
